ArduPlane/mode.h

```diff
@@ -395,6 +410,8 @@ class ModeLoiter : public Mode
     
     bool mode_allows_autotuning() const override { return true; }
 
+    bool requires_home() const override { return true; };
```

Comment:
Either make the method pure-virtual or rely on a default value from the base class.

Reply:
Done! I use the default from the base class, gps is required.